src/Yoke/Command.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Yoke\Command;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class YokeCommandCaller
+{
+    public $app;
+    public $input;
+    public $output;
+
+
+    public function __construct(Application $app, InputInterface $input, OutputInterface $output)
+    {
+        $this->app    = $app;
+        $this->input  = $input;
+        $this->output = $output;
+
+    }//end __construct()
+
+
+    public static function Create(Application $app, InputInterface $input, OutputInterface $output): YokeCommandCaller
+    {
+        return new YokeCommandCaller($app, $input, $output);
+
+    }//end Create()
+
+
+}//end class
+
+
+// TODO: Use YokeParameters instead of array
+class YokeParameters
```

Comment:
Woah! Didn't notice this. Well, I don't think it's cool to have more than one class per file.
Besides that I think it could be an interface or something.

Reply:
I think will broke the domain of the dependencies if we put it out. 

If somehow, is there a way to create all dependents a nested class of YokeCommand, will be great.

Refer: https://wiki.php.net/rfc/nested_classes - Someday we 'll see.